src/styles/_gradebook.scss

```diff
@@ -1,12 +1,20 @@
 .tiyo-assistant {
+    div.grades{
```

Comment:
I don't think we need `div` and `.grades` here, probably just the class. If there is a duplicate class name, let's just make it more specific.